src/frontend/src/flows/authorize/index.json

```diff
@@ -15,11 +15,12 @@
     "auth_failed": "Something went wrong during authentication. Authenticating service was notified and you may close this page.",
     "auth_success": "Authentication successful. You may close this page.",
 
+    "create_to_continue_to": "Create an Internet Identity to connect to",
     "to_continue_to": "to connect to",
     "is_alternative_of": "is an alternative domain of",
     "auth_same_identity": "and you will be authenticated to both with the same identity.",
 
-    "first_time_create": "Securely connect to dapps on the Internet Computer",
+    "first_time_create": "Securely connect with Internet Identity",
```

Comment:
asked my self if it should be phrased "Securely connect with an Internet Identity" so just sharing, just in case

Reply:
Product did consider it, then went for the current wording